templates/addons/linode-blockstorage-csi-driver-helm.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
+metadata:
+  name: linode-blockstorage-csi-driver
+spec:
+  clusterSelector:
+    matchLabels:
+      csi-driver: linode
+  repoURL: https://linode.github.io/linode-blockstorage-csi-driver/
+  chartName: linode-blockstorage-csi-driver
+  namespace: kube-system
+  options:
+    waitForJobs: true
+    wait: true
+    timeout: 5m
+  valuesTemplate: |
+    secretRef:
+      name: "linode-token-region"
+      apiTokenRef: "apiToken"
+      regionRef: "region"
```

Comment:
does the region have to be in the secret or can we keep just the token in the secret?

Reply:
good question, let me check

Reply:
yep, we can remove these two lines